internal/wrappers/build/android/app/src/main/jni/Android.mk

```diff
@@ -11,7 +11,7 @@ include $(CLEAR_VARS)
 LOCAL_MODULE    := kuzzle
 LOCAL_CFLAGS    := -frtti
 LOCAL_CPP_FEATURES := rtti exceptions
-LOCAL_CPPFLAGS += -I../../../../../../headers -I../../../../../../cpp/ -I../../../../../../build/c
+LOCAL_CPPFLAGS += -std=gnu++11 -I../../../../../../headers -I../../../../../../cpp/ -I../../../../../../build/c
```

Comment:
I would prefer that we activate standard C++11 extension only, without including GNU ones, for portability reasons.
Considering that we compile with standard C++11 in the general C++ makefile I won't make this remark stop the merging of this PR, but still, it would be cleaner...